endpoints/openrtb2/auction_benchmark_test.go

```diff
@@ -61,7 +64,7 @@ func BenchmarkOpenrtbEndpoint(b *testing.B) {
 	server := httptest.NewServer(http.HandlerFunc(benchmarkTestServer))
 	defer server.Close()
 
-	var infos config.BidderInfos
+	var infos = make(config.BidderInfos, 0)
```

Comment:
Does this change just initialize this variable?

Reply:
Exactly. Turns out the current version of this file in prod panics whenever we want to run this benchmark because the map is not explicitly initialized:
```
╭─gcarreongutierrez ~/go/src/github.com/prebid/prebid-server/endpoints/openrtb2  ‹master›
╰─➤  go test -run=XXX -bench=. -benchmem
goos: darwin
goarch: amd64
pkg: github.com/prebid/prebid-server/endpoints/openrtb2
cpu: Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
BenchmarkOpenrtbEndpoint-12     panic: assignment to entry in nil map

goroutine 50 [running]:
github.com/prebid/prebid-server/endpoints/openrtb2.BenchmarkOpenrtbEndpoint(0xc0000b6000)
        /Users/gcarreongutierrez/go/src/github.com/prebid/prebid-server/endpoints/openrtb2/auction_benchmark_test.go:65 +0x185
testing.(*B).runN(0xc0000b6000, 0x1)
        /usr/local/opt/go/libexec/src/testing/benchmark.go:193 +0x102
testing.(*B).run1.func1()
        /usr/local/opt/go/libexec/src/testing/benchmark.go:233 +0x59
created by testing.(*B).run1
        /usr/local/opt/go/libexec/src/testing/benchmark.go:226 +0x9c
exit status 2
FAIL    github.com/prebid/prebid-server/endpoints/openrtb2      0.441s
```
Explicitly initializing corrects this issue